spark-connector/spark-common/build.gradle.kts

```diff
@@ -56,24 +56,12 @@ dependencies {
   testCompileOnly(libs.lombok)
 
   // use log from spark, spark3.3 use low version of log4j, to avoid java.lang.NoSuchMethodError: org.apache.logging.slf4j.Log4jLoggerFactory: method <init>()V not found
-  testImplementation(project(":api")) {
-    exclude("org.apache.logging.log4j")
```

Comment:
why remove this? we should exclude log4j from Gravitino to avoid the conflict with Spark

Reply:
I rollback this change.